components/card/stories/template.js

```diff
@@ -188,7 +188,7 @@ export const Template = ({
           </div>`
       )}
       ${when(
-        footer && !isQuiet,
+        footer && !isQuiet && !isHorizontal && !isGallery,
```

Comment:
Great updates, thanks for clarifying these so that our settings are better reflecting the requirements from the designs.